source/globjects/source/implementations/TextureImplementation_DirectStateAccessEXT.cpp

```diff
@@ -34,6 +34,16 @@ void TextureImplementation_DirectStateAccessEXT::destroy(gl::GLuint id) const
     TextureImplementation_Legacy::instance()->destroy(id);
 }
 
+void TextureImplementation_DirectStateAccessEXT::bindActive(const Texture * texture, gl::GLuint unit) const
+{
+    gl::glBindTextureUnit(unit, texture->id());
```

Comment:
Unfortunately, the original [*direct state access* extension](https://www.khronos.org/registry/OpenGL/extensions/EXT/EXT_direct_state_access.txt) doesn't provide such an interface. We'll have to call the legacy implementation instead.